README.md

```diff
@@ -111,6 +132,84 @@ struct ExampleView: View {
 ```
 
 
+### Bulk Export of Historical Health Data
+
+The [`BulkHealthExporter`](https://swiftpackageindex.com/stanfordspezi/spezihealthkit/documentation/spezihealthkitbulkexport/bulkhealthexporter) enables export of historical HealthKit data through sessions that can process large amounts of data while maintaining memory efficiency and supporting resumption across app launches.
+
+First, configure the `BulkHealthExporter` in your `SpeziAppDelegate`:
+
+```swift
+import SpeziHealthKitBulkExport
+
+class ExampleAppDelegate: SpeziAppDelegate {
+    override var configuration: Configuration {
+        Configuration(standard: ExampleStandard()) {
+            HealthKit {
+                // ... your existing HealthKit configuration
+            }
+            BulkHealthExporter()
+        }
+    }
+}
+```
+
+Start the bulk export from your `Standard` after app launch:
+
+```swift
+actor ExampleStandard: Standard, HealthKitConstraint {
+    @Dependency(BulkHealthExporter.self) private var bulkExporter
+    
+    func configure() {
+        Task {
+            await performBulkExportIfNeeded()
+        }
+    }
```

Comment:
We should move this out of the configure method into the [`ServiceModule`](https://github.com/StanfordSpezi/Spezi/blob/main/Sources/Spezi/Capabilities/Lifecycle/ServiceModule.swift) setup that incorporates a proper usage of Swift concurrency: